tfhe/tests/zk_wasm_x86_test.rs

```diff
@@ -0,0 +1,86 @@
+//! Test compatibility between x86_64 and wasm proofs
+//!
+//! - Generate a crs and public key from rust
+//! - Load them in js, encrypt and prove some ciphertexts
+//! - Load the proven ciphertexts in rust and verify the proof
+
+#![cfg(feature = "zk-pok")]
+#![cfg(feature = "integer")]
+
+use std::fs::File;
+use std::path::{Path, PathBuf};
+use std::process::Command;
+use tfhe::safe_serialization::{safe_deserialize, safe_serialize};
+use tfhe::zk::{CompactPkeCrs, CompactPkePublicParams};
+use tfhe::{ClientKey, CompactPublicKey, ConfigBuilder, ProvenCompactCiphertextList};
+
+const SIZE_LIMIT: u64 = 1024 * 1024 * 1024;
+
+fn gen_key_and_crs() -> (CompactPublicKey, CompactPkeCrs) {
+    println!("Generating keys");
+    let config = ConfigBuilder::default().build();
+    let client_key = ClientKey::generate(config);
+    let pub_key = CompactPublicKey::new(&client_key);
```

Comment:
that's not the Blockchain config

see the test_proven_compact_list_with_casting

for config